### PR TITLE
[mlnx-fw-upgrade]: Define required FW version in build time.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**
- Do not query required FW from file.

### CODE REVIEW

File: files/build_templates/sonic_debian_extension.j2 [hunk 1]
```diff
@@ -223,6 +223,8 @@ sudo cp {{src}} $FILESYSTEM_ROOT/{{dst}}
 
 {% if sonic_asic_platform == "mellanox" %}
 sudo mkdir -p $FILESYSTEM_ROOT/etc/mlnx/
-sudo cp target/files/$MLNX_FW $FILESYSTEM_ROOT/etc/mlnx/fw-SPC.mfa
-sudo cp platform/mellanox/mlnx-fw-upgrade.sh $FILESYSTEM_ROOT/usr/bin/
+sudo cp target/files/$MLNX_FW_FILE $FILESYSTEM_ROOT/etc/mlnx/fw-SPC.mfa
+j2 platform/mellanox/mlnx-fw-upgrade.j2 > platform/mellanox/mlnx-fw-upgrade.sh
```
Comment (qiluo-msft) [inline files/build_templates/sonic_debian_extension.j2]: Why not redirect the stdout to target file under $FILESYSTEM_ROOT/usr/bin/?

Comment (oleksandrivantsiv) [inline files/build_templates/sonic_debian_extension.j2]: Linux will not allow it. To write something to image rootfs root permissions is required. There are two different operations: generation template with j2 and write it to file. With sudo will be executed only first one:
```
admin@arc-switch1026:~$ echo "OK" > /usr/bin/some_file
-bash: /usr/bin/some_file: Permission denied
admin@arc-switch1026:~$ sudo echo "OK" > /usr/bin/some_file
-bash: /usr/bin/some_file: Permission denied
admin@arc-switch1026:~$
```

Comment (qiluo-msft) [inline files/build_templates/sonic_debian_extension.j2]: I see your point. Could you try this command?
```
j2 platform/mellanox/mlnx-fw-upgrade.j2 | sudo tee $FILESYSTEM_ROOT/usr/bin/mlnx-fw-upgrade.sh
```

Comment (qiluo-msft) [inline files/build_templates/sonic_debian_extension.j2]: I see your point. Could you try this command?
```
j2 platform/mellanox/mlnx-fw-upgrade.j2 | sudo tee $FILESYSTEM_ROOT/usr/bin/mlnx-fw-upgrade.sh
```

**Pull request merged**